### PR TITLE
<RichTextInputArea /> - convert scss to stylable

### PR DESCRIPTION
**Pull request by yardenez**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6019

File: src/RichTextInputArea/RichTextInputArea.st.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
+@st-global-custom-property --wsr-color-30, --wsr-color-50, --wsr-color-10, --min-height, --max-height;
```
Comment (yardenez) [inline src/RichTextInputArea/RichTextInputArea.st.css:1]: Is this necessary? wasn't sure

File: src/RichTextInputArea/RichTextInputArea.st.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  background-color: value(backgroundColor);
+  border-radius: value(borderRadius);
+}
+
+.root :global(.public-DraftEditorPlaceholder-root){
+  padding: value(padding);
+  position: absolute;
+  pointer-events: none;
+  color:value(D40);
+  -st-mixin: medium font-mixin(family value(helveticanueu55));
```
Comment (EgoziE) [inline src/RichTextInputArea/RichTextInputArea.st.css:38]:
```suggestion
  -st-mixin: text-medium-normal;
```

File: src/RichTextInputArea/RichTextInputArea.st.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    background-color: value(backgroundColor);
+    min-height: var(--min-height, value(minHeight));
+    max-height: var(--max-height);
+    padding: value(padding);
+    border: value(border);
+    border-bottom-left-radius: value(borderRadius);
+    border-bottom-right-radius: value(borderRadius);
+    overflow: auto;
+    color:value(D10);
+    -st-mixin: medium font-mixin(family value(helveticanueu45))
```
Comment (EgoziE) [inline src/RichTextInputArea/RichTextInputArea.st.css:125]:
```suggestion
    -st-mixin: text-medium-thin;
```

File: src/RichTextInputArea/RichTextInputArea.st.css [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    background-color: value(D80);
+    cursor: text;
+    border-color: var(--wsr-color-10, value(B10));
+    box-shadow: 0 0 0 3px value(F00);
+}
+
+.link{
+  text-decoration: none; 
+  color:value(B10);
+  -st-mixin: medium font-mixin(family value(helveticanueu45))
```
Comment (EgoziE) [inline src/RichTextInputArea/RichTextInputArea.st.css:143]:
```suggestion
    -st-mixin: text-medium-thin;
```

File: src/RichTextInputArea/RichTextInputArea.st.css [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+:import {
+  -st-from: '../Foundation/stylable/colors.st.css';
+  -st-named: D80, B30, D10-30, R10, R30, Y10, B50, B10, F00,
+             D10, D40;
+}
+
+:import {
+  -st-from: '../Foundation/stylable/typography.st.css';
+  -st-named: medium, font-mixin, helveticanueu45, helveticanueu55;
```
Comment (EgoziE) [inline src/RichTextInputArea/RichTextInputArea.st.css:11]: Instead of all that, just import `text-medium-normal` and `text-medium-thin`

**Review by EgoziE — CHANGES_REQUESTED** [review]

**Review by EgoziE — APPROVED** [review]

**Pull request merged**